aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -1138,52 +1138,160 @@ void zeta_kernel(TensorIteratorBase& iter) {
   });
 }
 
+void jacobi_elliptic_k_cd_kernel(TensorIteratorBase& iterator) {
+ AT_DISPATCH_FLOATING_TYPES(iterator.common_dtype(), "jacobi_elliptic_k_cd_cpu", [&]() {
```

Comment:
Nice dispatching on common dtype to handle type promotion

Nice dispatch on just floating types, which is just float and double, so these templates are just instantiated for those dtypes